index.html

```diff
@@ -4,9 +4,167 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css">
     <title>Douglas' Portifolio page</title>
 </head>
 <body>
-    <h2>Portifolio</h2>
+<div class="main-layout">
```

Comment:
- Kindly use the same font styles and font weights as shown in the Figma Design for the whole webpage🤗
- Also please pay attention to the spacing between elements and the margin around them to make your website resemble the Figma design as closely as possible 😀

Your design:
![image](https://user-images.githubusercontent.com/73714615/186866139-94810258-a89a-44bb-93a1-b2ace646d190.png)

Figma Design:
![image](https://user-images.githubusercontent.com/73714615/186866183-8ab12d19-1f20-4e01-9d2c-528aa9185430.png)